### PR TITLE
chore: update readme file

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+  </a>
+</p>
 
-Logto sign-in experience sample codes
+[![discord](https://img.shields.io/discord/965845662535147551?color=5865f2&label=discord)](https://discord.gg/vRvwuwgpVX)
+[![cloud](https://img.shields.io/badge/cloud-available-7958ff)](https://cloud.logto.io/?sign_up=true&utm_source=github&utm_medium=repo_logto)
+
+# Logto sign-in experience UI samples
+
+Logto[^info] is an open-source Identity and Access Management (IAM) platform designed to streamline Customer Identity and Access Management (CIAM) and Workforce Identity Management. With Single Sign-On (SSO), OIDC-based authentication, and Multi-Tenant SaaS capabilities, Logto offers a scalable, secure, and developer-friendly way to manage authentication for modern web and mobile application.
```
Comment (gao-sun) [inline README.md:16]: can we use the same headline on the website?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,63 @@
...
+      <img width="200" src="https://github.com/logto-io/logto/raw/master/logo.png" alt="Logto logo">
+    </picture>
+  </a>
+</p>
 
-Logto sign-in experience sample codes
+[![discord](https://img.shields.io/discord/965845662535147551?color=5865f2&label=discord)](https://discord.gg/vRvwuwgpVX)
+[![cloud](https://img.shields.io/badge/cloud-available-7958ff)](https://cloud.logto.io/?sign_up=true&utm_source=github&utm_medium=repo_logto)
+
+# Logto sign-in experience UI samples
```
Comment (gao-sun) [inline README.md:14]: maybe

```suggestion
# Logto sign-in experience custom UI samples
```

**Review by gao-sun — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (charIeszhao) [reply README.md:14]: OK

File: README.md (hunk 1 above)
Comment (charIeszhao) [reply README.md:16]: You mean the official website?

**Pull request merged**